specs/lib/cacheOptimizeIndicesSpec.js

```diff
@@ -3,20 +3,31 @@ var cacheOptimizeIndices = require('../../lib/cacheOptimizeIndices');
 var addDefaults = require('../../lib/addDefaults');
 var readGltf = require('../../lib/readGltf');
 var readAccessor = require('../../lib/readAccessor');
+var writeAccessor = require('../../lib/writeAccessor');
 
-var gltfPath = './specs/data/cacheOptimizeIndices/chinesedragon.gltf';
+var gltfPath = './specs/data/boxTexturedUnOptimized/CesiumTexturedBoxTest.gltf';
+var optimizedIndices = [ 0, 1, 2, 3, 2, 1, 4, 5, 6, 7, 6, 5, 8, 9, 10, 11, 10, 9, 12, 
+    13, 14, 15, 14, 13, 16, 17, 18, 19, 18, 17, 20, 21, 22, 23, 22, 21 ];
 
 describe('cacheOptimizeIndices', function() {
     it('reorders indices', function(done) {
         readGltf(gltfPath, function(gltf) {
             addDefaults(gltf);
             var indexAccessorId = gltf.meshes[Object.keys(gltf.meshes)[0]].primitives[0].indices;
-            var indicesBefore = readAccessor(gltf, gltf.accessors[indexAccessorId]);
-            
+            var indexAccessor = gltf.accessors[indexAccessorId];
+            var indicesBefore = readAccessor(gltf, indexAccessor);
+            // Switch arbitrary pair of indices since CesiumTexturedBoxTest is already in cacheOptimized order
+            for (var i = 0; i < 3; i++ ) {
+                var temp = indicesBefore.data[i];
+                indicesBefore.data[i] = indicesBefore.data[i + 3];
+                indicesBefore.data[i + 3] = temp;
+            }
+            writeAccessor(gltf, indexAccessor, indicesBefore.data);
```

Comment:
Maybe instead of shifting the indices like this you can hard-code an array of `unoptimizedIndices` at the top.